src/nvim/generators/gen_options.lua

```diff
@@ -160,6 +160,9 @@ local dump_option = function(i, o)
     w('    .indir=PV_NONE')
     w('#endif')
   end
+  if o.cb then
+    w('    .opt_did_set_cb=' .. o.cb)
+  end
```

Comment:
Move this before the `#else` and revert the `#ifdef` change around `did_set_shellslash()`